mlflow/models/model.py

```diff
@@ -495,6 +495,10 @@ def to_dict(self):
             res["metadata"] = self.metadata
         if self.model_size_bytes is not None:
             res["model_size_bytes"] = self.model_size_bytes
+        if self.artifact_path is None:
```

Comment:
Could you add a comment for why artifact_path is not allowed? (so we don't remove it again:))

Reply:
added!